File: Mapper/Exception/ReadOnlyPropertyException.vb
Imports System
Imports System.Linq
Imports System.Reflection

Namespace Exception
    Public Class ReadOnlyPropertyException
        Inherits System.Exception

        Public Sub New([property] As PropertyInfo)

            MyBase.New("La propriété '" + [property].Name + "' de destination est en lecture seule")
        End Sub
    End Class
End Namespace

File: MapperCore/Exception/ReadOnlyPropertyException.cs
using System;
using System.Linq;
using System.Reflection;

namespace MapperCore.Exception
{
    public class ReadOnlyPropertyException :System.Exception
    {

        public ReadOnlyPropertyException(PropertyInfo property)
            : base("La propriété '" + property.Name + "' de destination est en lecture seule")
        {

        }
    }
}
